### PR TITLE
Add all initial migrations for DB.

### PR DESCRIPTION
**Pull request by KiriKaz**
Made [following the diagram.](https://dbdiagram.io/d/5f78c4773a78976d7b7638c8)

One note: Not exactly sure how the Reviews table is projected to work. It isn't really connected to products at all by foreign key - is it going to be purely through the meta in the product instance?

### CODE REVIEW

**Comment by KiriKaz** [comment]
I walked @MaiBaka through making a necessary change, one of the lines was off, so I walked him through checking out a branch, setting up his git with git config, and pushing the change he made.

File: database/migrations/2020_10_05_160539_create_addresses_table.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class CreateAddressesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('addresses', function (Blueprint $table) {
```
Comment (hoodedice) [inline database/migrations/2020_10_05_160539_create_addresses_table.php:16]: All fields except `id`, `contact_name` and the ones created by `->timestampsTz()` are nullable, please add on 
`->nullable()`

File: database/migrations/2020_10_05_162247_create_stores_table.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+class CreateStoresTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('stores', function (Blueprint $table) {
```
Comment (hoodedice) [inline database/migrations/2020_10_05_162247_create_stores_table.php:16]: let's set `description`, `url`, `address_id`, and `image` as nullable here

File: database/migrations/2020_10_05_163124_create_products_table.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+class CreateProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
```
Comment (hoodedice) [inline database/migrations/2020_10_05_163124_create_products_table.php:16]: `image`, `meta`, `warranty_text` should be nullable here

File: database/migrations/2020_10_05_170401_create_reviews_table.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+class CreateReviewsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reviews', function (Blueprint $table) {
```
Comment (hoodedice) [inline database/migrations/2020_10_05_170401_create_reviews_table.php:16]: `comment` is nullable here

**Review by hoodedice — CHANGES_REQUESTED** [review]

**Comment by hoodedice** [comment]
Good work so far @KiriKaz 
I think we'll let @MaiBaka deal with the changes I've requested. Once that's done I'll approve and we can merge these changes in

File: database/migrations/2020_10_05_162247_create_stores_table.php [hunk 5]
```diff
@@ -16,10 +16,10 @@ public function up()
         Schema::create('stores', function (Blueprint $table) {
             $table->id();
             $table->text('name');
-            $table->longtext('description');
-            $table->foreignId('address_id')->constrained();
-            $table->text('url');
-            $table->text('image');
+            $table->longtext('description')->nullable();
+            $table->foreignId('address_id')->nullable();
```
Comment (hoodedice) [inline database/migrations/2020_10_05_162247_create_stores_table.php]: You still need to keep the `->constrained()` at the end!
`$table->foreignId('address_id')->nullable()->constrained();`

**Review by hoodedice — CHANGES_REQUESTED** [review]

**Review by hoodedice — APPROVED** [review]
:+1: Good to merge

**Pull request merged**